### ISSUE TITLE
EM (Expectation Maximization) implementation

### ISSUE THREAD

**Issue by Rory-Finnegan** [invenia/Impute.jl#5]

**Comment by appleparan** [comment]
As I know, MICE, NN, EM methods are multivariate imputation. Is it possible to construct uniform API structure for these multivariate imputation method to this package?

**Comment by rofinn** [comment]
Yes, these methods would just be restricted to the corresponding input types (e.g., Matrices and Tables)
